launcher.py

```diff
@@ -286,7 +287,7 @@ def get_parser_args():
         raise ValueError('If --impute is present, then --phase must also be present')
 
     if args.command != 'reference' and args.use_bundle:
-        raise ValueError('--bundle option only available for reference downloading')
+        raise ValueError('--use-bundle option only available for reference downloading')
```

Comment:
Should it be here?

Reply:
Unfortunately yes. I fixed a bug that prevents downloading reference data as bundle.